### PR TITLE
added missing *.pb.h *.pb.cc generation to fix distribute build issue

### PR DESCRIPTION
**Pull request by putcn**
fix: #9309
looks grpc's cpp generation is missing the *.pb.h and *.pb.cc part. adding this back in this pr

related build error msg:
https://paddleci.ngrok.io/viewLog.html?buildId=31241&tab=buildLog&buildTypeId=Paddle_DistributedUnitTestNightly&logTab=tree&filter=all&_focus=7594

``` 
[07:33:06]	/paddle/build/paddle/fluid/operators/detail/send_recv.grpc.pb.cc:5:26: fatal error: send_recv.pb.h: No such file or directory
[07:33:06]	compilation terminated.
[07:33:06]	make[2]: *** [paddle/fluid/operators/detail/CMakeFiles/sendrecvop_grpc_grpc.dir/send_recv.grpc.pb.cc.o] Error 1
```

### CODE REVIEW

File: cmake/generic.cmake [hunk 1]
```diff
@@ -597,6 +597,9 @@ function(grpc_library TARGET_NAME)
           COMMAND ${PROTOBUF_PROTOC_EXECUTABLE}
           ARGS --grpc_out "${CMAKE_CURRENT_BINARY_DIR}" -I "${PROTO_PATH}"
           --plugin=protoc-gen-grpc="${GRPC_CPP_PLUGIN}" "${ABS_PROTO}"
+          COMMAND ${PROTOBUF_PROTOC_EXECUTABLE}
```
Comment (typhoonzero) [inline cmake/generic.cmake]: line 590 already calls `protobuf_generate_cpp(grpc_proto_srcs grpc_proto_hdrs "${ABS_PROTO}")`, curious why that doesn't work, and shoud line 590 be removed?

Comment (putcn) [reply cmake/generic.cmake]: hmm, i didn't notice this line, but this looks weird to me too. let me test if it's ok to remove the line 590.
also, I think it's good practice to put the generation command together, what's your idea?

Comment (putcn) [reply cmake/generic.cmake]: the build failed due to the missing of a dependency

```
The dependency target "sendrecvop_grpc_proto" of target "sendrecvop_grpc"
```

looks we still need to add line 590 back for now

Comment (typhoonzero) [reply cmake/generic.cmake]: If this just works, plz add a note here and merge this, we really need this unittest.

**Pull request merged**